filer/models/abstract.py

```diff
@@ -84,7 +89,7 @@ def _get_exif(self):
         if hasattr(self, '_exif_cache'):
             return self._exif_cache
         else:
-            if self.file:
+            if self.file.exists() and self.mime_type != 'image/svg+xml':
```

Comment:
Maybe we can just rewrite this by handling an exception in case `get_exif_for_file` fails? IDK if it can fail for other reason than file just being an svg

Reply:
There is an even nicer solution: `cached_property`.